blockchain/store_test.go

```diff
@@ -70,7 +69,7 @@ var (
 	part1       = partSet.GetPart(0)
 	part2       = partSet.GetPart(1)
 	seenCommit1 = &types.Commit{Precommits: []*types.Vote{{Height: 10,
-		Timestamp: time.Now().UTC()}}}
+		Timestamp: types.Now()}}}
```

Comment:
Maybe it will be better to call this `types.TimeNow()` or create a `time` subpackage, so this will be `tmtime.Now()`